satpy/scene.py

```diff
@@ -1143,6 +1143,45 @@ def save_datasets(self, writer=None, filename=None, datasets=None, compute=True,
                                           **kwargs)
         return writer.save_datasets(dataarrays, compute=compute, **save_kwargs)
 
+    def compute(self, **kwargs):
+        """Call `compute` on all Scene datasets.
```

Comment:
Should we say "data arrays" here instead of datasets to avoid the future confusion when Scene is more dependent on xarray Dataset objects?

Reply:
Yes definitely. I think this is better since this otherwise adds to confusion.